extensions/Worker.Extensions.EventGrid/src/TypeConverters/EventGridCloudEventConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
```

Comment:
If user configures JSON.NET as the JSON serializer, would using STJ for serialization here cause any issues? I remember there was an issue with SendGrid extension due to similar use case.

Reply:
I'm not sure but we can try and validate this scenario.

Reply:
validated no issues